### PR TITLE
update: finish up TODOs for cloud SIG

### PR DESCRIPTION
**Pull request by xibz**

### CODE REVIEW

File: sig-cloud/README.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ _OpsMx_
 
 
 ### Meeting Cadence
- - 1 hour, monthly, on **[Zoom](TODO-need-zoom-link)**
+ - 1 hour, monthly, on **[Google Meet](https://8x8.vc/greencapsdesirepolitely)**
```
Comment (dbyron-sf) [inline sig-cloud/README.md:59]: This doesn't look like a google meet link, but that probably doesn't matter.

**Review by dbyron-sf — APPROVED** [review]

File: sig-cloud/README.md (hunk 1 above)
Comment (link108) [reply sig-cloud/README.md:59]:
```suggestion
 - 1 hour, monthly, on **[Google Meet](https://meet.google.com/afb-rppx-zye)**
```

**Review by link108 — APPROVED** [review]
left a suggestion, but feel free to merge when ready!

File: sig-cloud/README.md (hunk 1 above)
Comment (xibz) [reply sig-cloud/README.md]: Ah yea, I think Google Meet uses 8x8 as the backend now, so the link's are basically the same thing, with different frontends.

**Pull request merged**